starlette/types.py

```diff
@@ -1,9 +1,17 @@
+import sys
 import typing
 
-Scope = typing.MutableMapping[str, typing.Any]
-Message = typing.MutableMapping[str, typing.Any]
+if sys.version_info >= (3, 10):  # pragma: no cover
+    from typing import TypeAlias
+else:  # pragma: no cover
+    from typing_extensions import TypeAlias
 
-Receive = typing.Callable[[], typing.Awaitable[Message]]
-Send = typing.Callable[[Message], typing.Awaitable[None]]
+Scope: TypeAlias = typing.MutableMapping[str, typing.Any]
```

Comment:
Is there a motivation for explicit type aliases here? Does the lack of TypeAlias cause issues? These changes seem unrelated to the goal of this PR (refining the `middleware_class: type` declaration), maybe we should consider a separate PR and discussion for this.

Reply:
Was discussed in a review by @Kludex above as well.

> See https://peps.python.org/pep-0613/. Here it does not add value, as it would be implicitly determined as a `TypeAlias` any way, but as `typing_extensions` is used in this project I figured why not use it, that way we know explicitly it is a `TypeAlias`.
I can revert this one as well

Reply:
Yep, please do. :pray: